documentation/source/autoindex.rst

```diff
@@ -7,21 +7,24 @@ autodoxygenindex Directive Example
 Working Example
 ---------------
 
-This should work::
+This should work
```

Comment:
It is probably too much to ask to go back and adjust all of these but `::` at the end of a sentence is rendered as a single `:` which is the intention here. So we're losing a `:` in each of these changes. I don't know if it is grammatically perfect but it makes sense to me to have one. We can revisit it another time though.

Reply:
Oh I forgot about that. That's my fault I'll fix that.

Reply:
I reverted this. Any text that ended with `::`  (& was replaced) is now a single `:`.

I also forced doxyfile snippets to be plain text as pygments inaccurately assumes that the doxyfile snippets are python syntax.